projects/angular/src/accordion/_accordion.clarity.scss

```diff
@@ -114,7 +114,7 @@
   .clr-accordion-header-has-description {
     .clr-accordion-title {
       min-width: accordion-variables.$clr-accordion-step-title-min-width;
-      max-width: mixins.baselinePx(264);
+      max-width: accordion-variables.$clr-accordion-step-title-min-width;
```

Comment:
I found issue with this hardcoded `min` and `max` width aproach. Check [here](https://1524--storybook-clarity-design.netlify.app/?path=/story/stepper-stepper--stepper&globals=viewport:small).

Description overflows.

![image](https://github.com/user-attachments/assets/f9bdf0e7-db1a-4208-9181-1dda57149ffe)

The better solution is to use percetnage based width.
Token for max width should be created as well.
Min width should not be changed.

```suggestion
      max-width: mixins.baselinePx(264);
      width: 100%;
```

Reply:
Thanks! This is fixed now.